front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -42,7 +48,45 @@ const ManchetteWithSpaceTimeChartWrapper = ({
   );
 
   const [showSettingsPanel, setShowSettingsPanel] = useState(false);
-  const [settings, setSettings] = useState({ showConflicts: false });
+  const [settings, setSettings] = useState({
+    showConflicts: false,
+    showBAL: false,
+    showBAPR: false,
+    showTVM: false,
+    showSignalsStates: false,
+  });
+
+  const occupancyBlocks = projectPathTrainResult
+    .filter((train) => train.signal_updates)
+    .flatMap((train) =>
+      train.signal_updates.flatMap((block) => ({
+        timeStart: +new Date(train.departure_time) + block.time_start,
+        timeEnd: +new Date(train.departure_time) + block.time_end,
+        spaceStart: block.position_start,
+        spaceEnd: block.position_end,
+        aspect_label: block.aspect_label,
```

Comment:
I don't remember this being part of the occupancy block type on the osrd-ui side, am I missing something?

(Same for `departure_time`)

Reply:
fixed